### DISCUSSION TITLE
i18n yaml file Localitation Editor

### DISCUSSION THREAD

**Discussion by d-oit** [gethinode/hinode#1142] [General]
What tools do you use to translate the i18n yaml files? I find tools that show whether a translation is missing helpful. 

For other projects I use
https://crowdin.com/ -> free for open source (export file is not valid ATM, support ticket created) 
POEditor
Own Json translation
RessourceEditor (Visual Studio)

**Comment by markdumay** [comment]
Thanks for the suggestions - both crowdin and POeditor look like helpful translation tools. So far it has been a manual effort, where various users have contributed an i18n file.
